antlr_config/ManimParser.g4

```diff
@@ -22,6 +22,8 @@ stat: SLEEP OPEN_PARENTHESIS expr CLOSE_PARENTHESIS SEMI                 #SleepS
     (ELSE OPEN_CURLY_BRACKET elseStat=stat? CLOSE_CURLY_BRACKET)?        #IfStatement
     | WHILE OPEN_PARENTHESIS whileCond=expr CLOSE_PARENTHESIS
       OPEN_CURLY_BRACKET whileStat=stat? CLOSE_CURLY_BRACKET             #WhileStatement
+    | FOR OPEN_PARENTHESIS forHeader CLOSE_PARENTHESIS
```

Comment:
Also minor but I don't think we need parentheses here at the moment. Maybe we can add them back if we extend for loops to have more complicated headers